void_migration/void_migration.py

```diff
@@ -248,7 +250,7 @@ def move_voids(
                 # if perf_plate and i+1==perf_pts[0]: P_r *= perf_rate
                 # if perf_plate and i+1==perf_pts[1]: P_r *= perf_rate
 
-                P_tot = P_u + P_l + P_r
+                P_tot = (0.98*P_u) + (0.98*P_l) + (0.98*P_r)
```

Comment:
what is this for? if you want more stability lower the stability cutoff in the latest version of void_migration